### ISSUE TITLE
Non-admin users can access Admin interface

### ISSUE THREAD

**Issue by z3ntu** [dularion/streama#32]
They can access the admin interface, either when they directly navigate to the url, or when they try to play an invalid format (not supported (btw thanks for the message!) dialog) and click okay there. Then they get redirected to the admin interface.

**Comment by dularion** [comment]
very true. I need to address this issue. But let me run something by you: I was thinking about using 2 roles, one called something like "contributor" and one called "admin." both contributors and admins can upload files, edit shows and movies etc, but only admins can add & edit users and change the settings. What do you think?

**Comment by z3ntu** [comment]
Would be nice! Also maybe a nicer name in the users overview, that the role is not "ROLE_ADMIN" but maybe just "Admin"!

**Comment by dularion** [comment]
:+1:

**Comment by dularion** [comment]
I'm not super happy with the name "contributor". Any ideas?

**Comment by z3ntu** [comment]
Uploader...?

**Comment by dularion** [comment]
Much better :)

**Comment by z3ntu** [comment]
Although it sounds a bit like pirating :100: (the people which upload torrents)

**Comment by dularion** [comment]
hehe true 
... hmmm
Maybe contributor isnt so bad after all

**Comment by z3ntu** [comment]
Sounds professional :)

**Comment by ChristianGaertner** [comment]
"Content Manager" sounds cool as well.

**Comment by dularion** [comment]
"Content Manager" :+1:

**Comment by dularion** [comment]
All right, so with the last few commits I made the following adjustments: 
- added "displayName" for roles
- added new Role "Content Manager"
- server-side lockdown of several REST-endpoints, so that they are only accessible by certain roles
- edit-dialog for users improved: added role-management


And these are the basic rules for the two roles (I added this info to the GUI as well)
Users with the role **Admin** can edit Users & Settings. 
Users with the role **Content Manager** can edit content. 
Users with **both rules** have full rights.


It would be great if someone other than me tested all of this to see if it works (you might test differently from me and find some hidden bugs)
